packages/yourloops/components/menus/team-menu.tsx

```diff
@@ -169,24 +191,30 @@ function TeamMenu(): JSX.Element {
           </ListItem>
         }
 
-        {isUserHcp &&
+        {isUserHcp || isUserPatient &&
           <Box>
             <Box marginY={1}>
               <Divider variant="middle" />
             </Box>
 
-            <MenuItem id="team-menu-teams-link" onClick={createCareTeam}>
+            <MenuItem id="team-menu-teams-link" onClick={onTeamMenuTeamsAction}>
```

Comment:
We are in the component team-menu, could be simplified
```suggestion
            <MenuItem id="team-menu-teams-link" onClick={onTeamAction}>
```